### DISCUSSION TITLE
[retext-spell] Multiple suggestions based on personal dictionary

### DISCUSSION THREAD

**Discussion by vincentmvdm** [retextjs/retext#67] [Q&A] [answered]
I'm adding a word to my personal dictionary that has two correct spellings: one where only the first letter is capitalized, and one that's fully capitalized. For example, 'Lorem' and 'LOREM' are both correct, but 'lorem' is incorrect.

Whenever I write 'lorem', the spellchecker only suggests 'Lorem' however. Is there a way to make it suggest both 'Lorem' and 'LOREM' in that instance?

Thank you! I absolutely love this package.

**Comment by wooorm** [comment, marked as answer]
Interesting! Hadn’t thought of that. The code I wrote only supports one case: https://github.com/wooorm/nspell/blob/main/lib/util/form.js. You could report it on nspell if you think that’s wrong! (but that project is rather low on my todo list currently, so I’m not getting to it soon)
